adcc/ExcitedStates.py

```diff
@@ -237,6 +241,34 @@ def transition_dipole_moments(self):
             for tdm in self.transition_dms
         ])
 
+    @cached_property
+    @timed_member_call(timer="_property_timer")
+    def transition_velocity_dipole_moments(self):
+        """List of transition dipole moments in the
+        velocity gauge of all computed states"""
+        if self.property_method.level == 0:
+            warnings.warn("ADC(0) transition velocity dipole moments "
+                          "are known to be faulty in some cases.")
```

Comment:
I don't know if they're faulty, but since we don't have reference values from `adcman`, a warning is better than nothing, right? I just copied it for all following case as well.

Reply:
By the way is this not also tested by the CIS tests you're coding now? Maybe that would allow us to finally resolve this? If you have the time (and everything setup somewhere) could you just test the offending case? I think it should show when you remove the pytest skip from the repective property / functionality test.